src/Nordea/Resources/Fonts.elm

```diff
@@ -9,7 +9,7 @@ fromSize : Float -> Style
 fromSize size =
     let
         fontFamily =
-            if size < 1.125 then
+            if size <= 1.375 then
```

Comment:
Hmm, this seems ambiguous... both fonts can be used for 22px

![image](https://user-images.githubusercontent.com/1189327/144867880-88f62ccb-d5fa-442f-b3bb-1b13951ab18e.png)

Reply:
But anyway, 1.375rem is more correct than 1.125rem 😅

Reply:
nvm lol